app/models/shelter.rb

```diff
@@ -4,6 +4,8 @@ class Shelter < ApplicationRecord
   validates :city, presence: true
 
   has_many :pets, dependent: :destroy
+  has_many :application_pets
```

Comment:
A shelter doesn't have a link to application_pets, so this association should not be here.